types/sparql-http-client/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{ 
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
```

Comment:
Is there a reason for turning off this rule? I see that the npm package has the same name.

Reply:
Well yes, this particular rule gives the "default export" error which you also pointed out

Reply:
This rule is unrelated to the `default`. This is for the package name and the d.ts name which I don't see they should have a problem.